internal/saucecloud/retry/project.go

```diff
@@ -0,0 +1,7 @@
+package retry
+
+import "github.com/saucelabs/saucectl/internal/saucereport"
+
+type Project interface {
+	FilterFailedTests(suiteIndex int, report saucereport.SauceReport) error
```

Comment:
Why the preference for the index, rather than the name (or something else that's concrete)? Just curious. It's a fairly loose identifier and doesn't carry enough information if something were to go wrong (index out of bounds).

Reply:
The original thoughts is for handling sharding. Now I think it's fine to use suite name to filter out the sutie.